packages/logs/src/rawLogsEvent.types.ts

```diff
@@ -0,0 +1,34 @@
+import type { Context, ErrorSource, TimeStamp } from '@datadog/browser-core'
+import type { StatusType } from './domain/logger'
+
+export interface RawLogsEvent {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could it be interesting to document the different raw types, console, report, logger, ...?

Reply:
Indeed it could be nice to have.